tests/Doctrine/Tests/ORM/Decorator/EntityManagerDecoratorTest.php

```diff
@@ -52,7 +51,12 @@ private function getParameters(\ReflectionMethod $method)
 
         /** Special case EntityManager::transactional() */
         if ($method->getName() === 'transactional') {
-            return [$method->getName(), [function () {}]];
+            return [
+            $method->getName(),
+            [function () {
```

Comment:
This is incorrectly indented, can we find out which sniff did this and report it?